runtime/onert/core/src/compiler/ExecutorFactory.cc

```diff
@@ -217,6 +239,33 @@ createBackendContexts(compiler::ILoweredGraph &lgraph, bool linear_executor,
 
   // Create contexts
   auto whole_op_order = lgraph.graph().topolSortOperations();
+  const std::unordered_set<std::string> memory_sharing_supported_backends = {"cpu", "builtin"};
```

Comment:
Please remove `builtin` backend from this set. The kernels in `builtin` backend deal with data transmission between other backends, so there is no need to apply in-place for this task. The required in-place optimization there has already been applied in the other way.

Reply:
I see - I re-wrote implementation to be local for cpu.